notifier/notifications/notifications.go

```diff
@@ -85,11 +86,19 @@ func (worker *FetchNotificationsWorker) processScheduledNotifications() error {
 			}
 		}
 		p.Events = append(p.Events, notification.Event)
+
+		err = worker.Database.PushContactNotificationToHistory(notification)
+
+		if err != nil {
+			worker.Logger.Warning().Msg("Can't save notification to history" + err.Error())
```

Comment:
У нас принято использовать структурированные логи:
```suggestion
			worker.Logger.Warning().Error(err).Msg("Can't save notification to history")
```